src/main/java/core/basesyntax/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportService;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class ReportServiceImpl implements ReportService {
+    private static final String REPORT_TITLE = "fruit,quantity" + System.lineSeparator();
```

Comment:
```suggestion
    private static final String REPORT_TITLE = "fruit,quantity";
```
use line separator in generate report when needed, but it has nothing to do with this constant